src/client/api/pythonApi.ts

```diff
@@ -49,13 +49,32 @@ import {
 @injectable()
 export class PythonApiProvider implements IPythonApiProvider {
     private readonly api = createDeferred<PythonApi>();
+    private readonly didActivatePython = new EventEmitter<void>();
```

Comment:
Do we need to make sure we dispose of this object?

Reply:
Fixed.